backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/MolgenisSession.java

```diff
@@ -14,16 +13,14 @@
 public class MolgenisSession {
   private static final Logger logger = LoggerFactory.getLogger(MolgenisSession.class);
 
-  private String sessionUser;
+  private String token;
   private Database database;
   private GraphQL graphqlForDatabase;
-  private DateTime createTime;
   private Map<String, GraphQL> graphqlPerSchema = new LinkedHashMap<>();
 
-  public MolgenisSession(Database database) {
+  public MolgenisSession(Database database, String token) {
+    this.token = token;
```

Comment:
remove token field here, the MolgenisSession works fine without it

Reply:
agreed